MatterStore/recipeStore
# MS-85: Recipe Edit UI

> [MatterSpace PBI](https://matterstore.atlassian.net/browse/MS-)

## This PR (Introduces / Fixes)
Introduces front-end only editing of recipes. It does not yet including modification of recipe images or tags, or publicity, though the design for such is partially implemented.

## To test this PR
Open a recipe, attempt to mutate it's name, servings, steps, ingredients. Toggle the mode of the step and ingredient list text area, see if it behaves as expected. See if clicking save shows the corrected mutated state.

**Release Note:**
- Allow front end editing of recipes

## Demonstrative/process videos

https://user-images.githubusercontent.com/8501694/191217069-07164215-cdf4-4a35-81f5-545f19e30053.mp4



https://user-images.githubusercontent.com/8501694/191217040-cf00dce8-c3c0-4b27-ad49-d24fd8776724.mp4


https://user-images.githubusercontent.com/8501694/191217057-f699ef84-cec0-41a5-ad10-b44ad66ac6be.mp4
